aea/configurations/loader.py

```diff
@@ -96,12 +96,16 @@ def load_protocol_specification(self, file_pointer: TextIO) -> T:
         for document in yaml_data:
             yaml_documents.append(document)
         configuration_file_json = yaml_documents[0]
-        if len(yaml_documents) == 2:
-            protobuf_snippets_json = yaml_documents[1]
-        elif len(yaml_documents) == 1:
+        if len(yaml_documents) == 1:
             protobuf_snippets_json = {}
+            dialogue_configuration = {}  # type: Dict
+        elif len(yaml_documents) > 1:
```

Comment:
To be safe, let's do `len(yaml_documents) in [2,3]`

Reply:
Done